wdl-analysis/src/util.rs

```diff
@@ -0,0 +1,39 @@
+//! Utility functions for wdl-analysis.
+
+// Add any additional imports needed
+
+/// Iterates over the lines of a string and returns the line, starting offset,
+/// and next possible starting offset.
+pub fn lines_with_offset(s: &str) -> impl Iterator<Item = (&str, usize, usize)> {
```

Comment:
instead of a new `util` module, just put this in `wdl-analysis/src/lib.rs` and remove it from `wdl-lint`. Then update all the uses of it in `wdl-lint` to import from `wdl-analysis` instead